src/GTranslate/LanguageDictionary.cs

```diff
@@ -142,153 +142,167 @@ public bool TryGetLanguage(string code, [MaybeNullWhen(false)] out Language lang
         ["az"] = new("Azerbaijani", "Azərbaycan", "az", "aze"),
-        ["eu"] = new("Basque", "Euskara", "eu", "eus"),
+        ["ba"] = new("Bashkir", "Bashkir", "ba", "bak", TranslationServices.Bing | TranslationServices.Microsoft | TranslationServices.Yandex),
         ["be"] = new("Belarusian", "беларуская", "be", "bel", TranslationServices.Google | TranslationServices.Yandex),
+        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
+        ["bho"] = new("Bhojpuri", "भोजपुरी", "bho", "bho", TranslationServices.Google),
+        ["bm"] = new("Bambara", "ߓߊߡߊߣߊ߲ߞߊ߲", "bm", "bam", TranslationServices.Google),
         ["bn"] = new("Bengali", "বাংলা", "bn", "ben"),
+        ["bo"] = new("Tibetan", "བོད་སྐད་", "bo", "bod", TranslationServices.Bing | TranslationServices.Microsoft),
         ["bs"] = new("Bosnian", "bosanski", "bs", "bos"),
-        ["bg"] = new("Bulgarian", "Български", "bg", "bul"),
-        ["my"] = new("Burmese", "မြန်မာ", "my", "mya"),
         ["ca"] = new("Catalan", "Català", "ca", "cat"),
         ["ceb"] = new("Cebuano", "Binisaya", "ceb", "ceb", TranslationServices.Google | TranslationServices.Yandex),
-        ["ny"] = new("Chichewa", "Nyanja", "ny", "nya", TranslationServices.Google),
-        ["zh-CN"] = new("Chinese (Simplified)", "中文 (简体)", "zh-CN", "zho-CN"),
-        ["zh-TW"] = new("Chinese (Traditional)", "繁體中文 (繁體)", "zh-TW", "zho-TW", TranslationServices.Google | TranslationServices.Bing | TranslationServices.Microsoft),
         ["co"] = new("Corsican", "Corsu", "co", "cos", TranslationServices.Google),
-        ["hr"] = new("Croatian", "Hrvatski", "hr", "hrv"),
         ["cs"] = new("Czech", "Čeština", "cs", "ces"),
+        ["cv"] = new("Chuvash", "Чӑвашла", "cv", "chv", TranslationServices.Yandex),
+        ["cy"] = new("Welsh", "Cymraeg", "cy", "cym"),
         ["da"] = new("Danish", "Dansk", "da", "dan"),
-        ["nl"] = new("Dutch", "Nederlands", "nl", "nld"),
+        ["de"] = new("German", "Deutsch", "de", "deu"),
+        ["dgo"] = new("Dogri", "डोगरी (ḍogrī) ڈوگرى", "dgo", "dgo", TranslationServices.Google),
+        ["dv"] = new("Divehi", "ދިވެހިބަސް", "dv", "div", TranslationServices.Bing | TranslationServices.Microsoft),
+        ["ee"] = new("Ewe", "Eʋegbe", "ee", "ewe", TranslationServices.Google),
+        ["el"] = new("Greek", "Ελληνικά", "el", "ell"),
+        ["emj"] = new("Emoji", "Emoji", "emj", "emj", TranslationServices.Yandex), // Not present in Yandex.Cloud
         ["en"] = new("English", "English", "en", "eng"),
+        ["en-GB"] = new("English (UK)", "English (UK)", "en-GB", "en-GB", TranslationServices.Google),
         ["eo"] = new("Esperanto", "Esperanto", "eo", "epo", TranslationServices.Google | TranslationServices.Yandex),
+        ["es"] = new("Spanish", "Español", "es", "spa"),
+        ["es-MX"] = new("Spanish (Mexico)", "español (México)", "es-MX", "es-MX", TranslationServices.Bing | TranslationServices.Microsoft),
```

Comment:
Again, can you please provide more details as to how es is treated as the same as es-MX?

I just ran a test to translate the word "**Computer**".
* In "es" I got the following results:
  * Computadora
* In "es-MX" I got the following results:
  * Ordenador

This is using GTranslate to translate the word.  Unless you know the language differences, the translations can appear identical because 99% of the translations will be the same.  But a few words are different.

Reply:
That's happening because the result from `es` doesn't come from Bing/Microsoft translator.

If you modify line 66 of AggregateTranslator.cs so that the only translator present is `MicrosoftTranslator`..
```diff
- _translators = new ITranslator[] { googleTranslator, googleTranslator2, microsoftTranslator, yandexTranslator, bingTranslator };
+ _translators = new ITranslator[] { microsoftTranslator };
```
..and then use `AggregateTranslator` through the example program, you will notice that both `es` and `es-MX` return the same result.

![image](https://user-images.githubusercontent.com/39978852/188334212-20c52a0a-1c80-4134-99bd-36941d83837b.png)

If you try to do this with `BingTranslator` you'll get a 400 Bad Request status code.

Reply:
From my test, the correct translation is coming from TranslationServices.Microsoft.

TranslationServices.Bing fails when it's the only one listed as a translation service.  Which means the correct change should be the following:
`["es-MX"] = new("Spanish (Mexico)", "español (México)", "es-MX", "es-MX", TranslationServices.Microsoft),`

If it works with above values, why remove it?

Reply:
> If it works with above values, why remove it?

Because it's misleading, there isn't a `es-MX` language code listed in the docs, and the API treats `es` and `es-MX` as the same language because the API returns an `es` target language as you can see in the image above. You can even get the same result by creating a fake language code like `es-LL` or `en-KO`. This means the API just disregards anything after the "-" when it's not required.

Reply:
> Because it's misleading, there isn't a es-MX language code listed in the docs, and the API treats es and es-MX as the same language because the API returns an es

I was confused before because when you said API, I thought you were referring to GTranslate API.  From above comment, it seems you're referring to the translation service API.

With that said, I thought you're saying it's misleading because the translation service returns "es" in its response. But when I tested this out, I don't see "es" by itself any where in the response returned by GTranslate API.

Here's the code calling GTranslate API:
`GTranslate.Results.ITranslationResult result = await translator.TranslateAsync("Computer", "es-MX", "en");`

Here's the content of the result variable:
<img width="783" alt="es-MX_Result" src="https://user-images.githubusercontent.com/47364845/188486127-b895480c-e276-4828-86f7-c7c0c4926c5c.png">

GTranslate API results are **NOT** misleading with the input vs the output.

Reply:
That's easy to explain. In line 219 of MicrosoftTranslator.cs you'll notice that the method returns `Language.GetLanguage(toLanguage.ISO6391)`. This means the method ignores the target language returned from the API response because I expected Microsoft Translator would return an error code when an invalid language code is passed instead of just cutting the extra part.

Reply:
> This means the method ignores the target language returned from the API response 

That sounds good. That means the code will work as-is with es-MX.

I still don't understand the justification for removing it, if it fully works.

Reply:
I already mentioned why [here](https://github.com/d4n3436/GTranslate/pull/3#discussion_r962540851).